app/views/spree/shared/_payment.html.haml

```diff
@@ -13,3 +13,10 @@
     = t :email_payment_summary
 - if @order.payments.any?
   = render partial: 'spree/shared/payments_list', locals: { payments: @order.payments }
+
+- if @order.payments.any?
+  %p.callout{style: "margin-top: 40px"}
+    %strong
+      = t :email_payment_description
```

Comment:
According to the issue, it seems that we used to use the payment method name as the header. We could check it out in the git history.

Reply:
I can see @kirstenalarsen comment saying: 
> Under Payment Summary there is a field called Payment Description at Checkout: which then prints the description info, not in the table just width of the page